client/src/components/AttractionOverview.jsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
```

Comment:
Prolly don't want to merge commented sections in to main branch. We can maintain a side branch for these things in progress. The more branches the better.